addons/base_import_module/models/ir_module.py

```diff
@@ -25,6 +25,11 @@ class IrModule(models.Model):
     _inherit = "ir.module.module"
 
     imported = fields.Boolean(string="Imported Module")
+    module_type = fields.Selection([
+        ('official', 'Official Apps'),
```

Comment:
At the cost of needing an upgrade script, we can merge the values.
```suggestion
    module_type = fields.Selection([
        ('official', 'Official Apps'),
        ('imported', 'Imported Data'),
```